OpenRA.Mods.Common/Widgets/Logic/MusicPlayerLogic.cs

```diff
@@ -18,29 +18,24 @@ namespace OpenRA.Mods.Common.Widgets.Logic
 {
 	public class MusicPlayerLogic
 	{
-		readonly Ruleset modRules;
-
-		bool installed;
 		MusicInfo currentSong = null;
-		MusicInfo[] music;
-		MusicInfo[] random;
-		ScrollPanelWidget musicList;
+		MusicPlaylist musicPlaylist;
```

Comment:
Readonly here and for the two widgets below please.  The general rule is that if it's set in a constructor and isn't supposed to change, then it should be marked readonly.